server/src/main/java/fr/uga/l3miage/tp4/entities/CandidateEvaluationGridEntity.java

```diff
@@ -0,0 +1,26 @@
+package fr.uga.l3miage.tp4.entities;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CandidateEvaluationGridEntity {
+    @Id
+    private Long sheetNumber;
+    private Double grade;
+    private LocalDateTime submissionDate;
+    @ManyToOne
+    private ExamEntity examEntity;
+    @ManyToOne
+    private CandidateEntity candidateEntity;
```

Comment:
les critères d'evalutation on plus de sens à être dans la grid d'evaluation que le contraire, n'hésitez pas à prendre un peu de recul 😅 
```suggestion
    private CandidateEntity candidateEntity;
    
    @ManyToMany
    private Set<EvaluationCriteriaEntity> evaluationCriteriaEntities;
```